### ISSUE TITLE
[Counter-Strike: Source] LAN Servers are not shown in in-game server browser

### ISSUE THREAD

**Issue by NextBlaubeere** [otavepto/gbe_fork#182]
When setting up Dedicated LAN Servers, they're not shown in the in-game server browser.

They are shown on the legit steam version.

I am able to join the server with gbe and steam version of the game via console "connect <IP>".

I did not found anything in the settings files regarding LAN server browsers.

Legit Steam
![Steam](https://github.com/otavepto/gbe_fork/assets/10467690/009e5848-cde9-4f3e-ad1d-e758026200e6)

GBE
![GBE](https://github.com/otavepto/gbe_fork/assets/10467690/80907a83-c3d0-40fc-8eff-8b7383d4dc32)

**Comment by NextBlaubeere** [comment]
Those are for Counter-Strike GO/2, aren't they?

My question was regarding Counter-Strike _Source_ or Source games in genereal. (Day of Defeat Source, Half Life 2 Deathmatch, ...)

**Comment by M4RCK5** [comment]
idk if its the same like in goldsrc games but in the older ones u had to use "sv_lan 0"

**Comment by GinDes** [comment]
yeah, I found the same problem with valheim, I can't find it in the server browser. I tried the latest, but couldn't find the server.
